plugins/org.eclipse.xtend.ide.common/src/org/eclipse/xtend/ide/common/highlighting/XtendHighlightingCalculator.java

```diff
@@ -384,6 +384,7 @@ public void acceptSemanticLineBreak(int charCount, RichStringLiteral origin, boo
 		@Override
 		public void acceptTemplateLineBreak(int charCount, RichStringLiteral origin) {
 			resetCurrentOffset(origin);
+			acceptor.addPosition(currentOffset, charCount, TEMPLATE_LINE_BREAK);
```

Comment:
Since we do the highlighting in IDEA the other way round as in eclipse, I don't see how we can reuse the entire implementation of the richstring highlighter.